sql/moz-fx-data-shared-prod/monitoring_derived/shredder_rows_deleted_v1/schema.yaml

```diff
@@ -0,0 +1,21 @@
+- name: end_time
+  type: TIMESTAMP
+  mode: REQUIRED
+- name: job_id
+  type: STRING
+  mode: REQUIRED
+- name: project_id
+  type: STRING
+  mode: REQUIRED
+- name: dataset_id
+  type: STRING
+  mode: REQUIRED
+- name: table_id
+  type: STRING
+  mode: REQUIRED
+- name: partition_id
+  type: STRING
+  mode: NULLABLE
+- name: deleted_row_count
```

Comment:
Daniel, is there a way we could calculate in this table the `client_ids` deleted?

Reply:
doing so would require that we query each table twice, to count the client ids in it before and after, something that would likely incur considerable cost.

I think it would be better to approximate the number of client ids deleted by checking how many rows were deleted from an appropriate clients daily type table.